web/content/3.0.md

```diff
@@ -36,9 +36,51 @@ plugins. Official documentation [is available here](https://theforeman.org/manua
 
 ### Katello headline features
 
+For the full list of changes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-4.2/CHANGELOG.md).
+
+* New Experimental Content View UI Features:
+  * Publish & Promote
+  * Add component CVs to composite CVs
+* Pulp's mirorred metadata is now used to speed up repo syncs
+* Upgrade to Pulp 3.14 (This was backported to Katello 4.1)
+* support deleting RH repos from the repositories details
+
+## Upgrade Warnings
+
+* upstream_subscriptions apis have been removed in favor of apis rooted at /katello/api/v2/organizations/:organization_id/simple_content_access/
+
 ## Deprecations
 
 ### Contributors to Foreman 3.0
 
 ### Contributors to Katello 4.2
 
+* Andrew Dewar
+* Andrew Lukoshko
+* Bernhard Suttner
+* Chris Roberts
+* Dominik Matoulek
+* Eric D. Helms
+* Hao Yu
+* Ian Ballou
+* James Jeffers
+* Jeremy Lenz
+* Jonathon Turel
+* Justin Sherrill
+* Leos Stejskal
+* Lucy Fu
+* Lukas Zapletal
+* Marek Hulán
+* Markus Bucher
+* Oleh Fedorenko
+* Ondřej Ezr
+* Partha Aji
+* Pat Riehecky
+* Paul Donohue
+* Ryan Verdile
+* Samir Jha
+* Stejskal Leos
+* lumarel
+* rverdile
```

Comment:
I think this is Ryan Verdile which would be a duplicate of line 80. If you're using git to collect these, see https://github.com/theforeman/foreman-documentation/commit/96b99ff2e900845eea976aee2e969cc24c7aedcc for an example of how to map these.

Reply:
that command returns rverdile too, i think he probably had a misconfigured git client at some point too